spec/performable_method_spec.rb

```diff
@@ -1,9 +1,10 @@
 require 'helper'
+require 'action_controller/metal/strong_parameters' if ActionPack::VERSION::MAJOR >= 5
 
 describe Delayed::PerformableMethod do
   describe 'perform' do
     before do
-      @method = Delayed::PerformableMethod.new('foo', :count, ['o'])
```

Comment:
@korny @NourEldinShobier I haven't yet figured out why they are passing ['o'] here. Since I don't think it's necessary, I removed the square brackets.

Reply:
As explained, the brackets are necessary as this needs to be an Array.